src/Fable.Cli/ProjectCracker.fs

```diff
@@ -459,17 +471,20 @@ let createFableDir (opts: CrackerOptions) =
 
     fableDir
 
+let copyDir (source: string) (target: string) =
+    IO.Directory.CreateDirectory(target) |> ignore
+    if IO.Directory.Exists source |> not then
+        failwith ("Source directory is missing: " + source)
+    let source = source.TrimEnd('/', '\\')
+    let target = target.TrimEnd('/', '\\')
+    for dirPath in IO.Directory.GetDirectories(source, "*", IO.SearchOption.AllDirectories) do
+        IO.Directory.CreateDirectory(dirPath.Replace(source, target)) |> ignore
+    for newPath in IO.Directory.GetFiles(source, "*.*", IO.SearchOption.AllDirectories) do
+        IO.File.Copy(newPath, newPath.Replace(source, target), true)
+
 let copyDirIfDoesNotExist (source: string) (target: string) =
     if isDirectoryEmpty target then
-        IO.Directory.CreateDirectory(target) |> ignore
-        if IO.Directory.Exists source |> not then
-            failwith ("Source directory is missing: " + source)
-        let source = source.TrimEnd('/', '\\')
-        let target = target.TrimEnd('/', '\\')
-        for dirPath in IO.Directory.GetDirectories(source, "*", IO.SearchOption.AllDirectories) do
-            IO.Directory.CreateDirectory(dirPath.Replace(source, target)) |> ignore
-        for newPath in IO.Directory.GetFiles(source, "*.*", IO.SearchOption.AllDirectories) do
-            IO.File.Copy(newPath, newPath.Replace(source, target), true)
+        copyDir source target
 
 let copyFableLibraryAndPackageSources (opts: CrackerOptions) (pkgs: FablePackage list) =
```

Comment:
@alfonsogarciacaro If you think this is too messy with all the language matches, I could perhaps make a separate `copyFableLibraryAndPackageSourcesPy` function and have match before calling!?

Reply:
Added a separate copy function for Python to avoid the 3 messy language matches